src/feature_flags.ts

```diff
@@ -3,9 +3,9 @@ import { env } from 'process'
 export const defaultFlags: Record<string, boolean> = {
   buildGoSource: Boolean(env.NETLIFY_EXPERIMENTAL_BUILD_GO_SOURCE),
   buildRustSource: Boolean(env.NETLIFY_EXPERIMENTAL_BUILD_RUST_SOURCE),
-  defaultEsModulesToEsbuild: Boolean(env.NETLIFY_EXPERIMENTAL_DEFAULT_ES_MODULES_TO_ESBUILD),
   parseWithEsbuild: false,
   traceWithNft: false,
+  zisi_detect_esm: false,
```

Comment:
This determines whether we'll look at a `.js` file and figure out whether it's CJS or ESM.